src/components/sections/ShopLinks/ShopLinks.tsx

```diff
@@ -0,0 +1,33 @@
+// libs imports
+import React from 'react';
+// local imports
+import { Section, Container } from 'components/common';
+import { ShopLink } from './ShopLink';
+import css from './ShopLinks.module.scss';
+import { links } from 'data';
+
+const ShopLinks: React.FC = (): JSX.Element => {
+  return (
+    <Container>
+      <Section className={css.section}>
+        <ul className={css.list}>
+          {links.map(link => {
+            const { name, images, target } = link;
```

Comment:
You can just pass the whole object into the component props, which is more efficient. 

But it is OK as well.

Reply:
fixed on another branch - https://github.com/myroslavsuprun/audiophile/tree/feature/form-elements